jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -55,52 +55,6 @@
     - 'ci-{suffix}'
     name: bootstrap-ci
     suffix:
-    # gce high-scale tests #shyamjvs
```

Comment:
Can we gradually roll these jobs instead of all at once? We might face some issues on these huge scale jobs when running on prow - and can't afford to lose all our runs at once.

I'd suggest moving the following for now:
- gke-(large|scale)-correctness
- gke-large-performance
- gce-large-(performance|correctness)

Leaving out the following crucial release-blocking ones:
- gce-scale-(performance|correctness)

Reply:
ack, will update